src/lib.rs

```diff
@@ -143,10 +143,10 @@ use users::Users;
 const DEFAULT_HOST: &str = "https://api.github.com";
 
 /// A type alias for `Futures` that may return `hubcaps::Errors`
-pub type Future<T> = Box<StdFuture<Item = T, Error = Error>>;
+pub type Future<T> = Box<StdFuture<Item = T, Error = Error> + Send>;
```

Comment:
Thank you for adding this! I had a branch based on this one adding this because I needed it myself; I was going to wait for this PR before submitting it.

Reply:
yeah, it's actually required when using `tokio::runtime::Runtime` or `tokio::runtime::current_thread::Handle`